extensions/3DTILES_multiple_contents/0.0.0/README.md

```diff
@@ -19,21 +20,44 @@ Draft
 
 Written against the 3D Tiles 1.0 specification.
 
+Adds new functionality to the [`3DTILES_implicit_tiling` extension](../../3DTILES_implicit_tiling/README.md). See [Implicit Tiling](#implicit-tiling)
+
+<!-- omit in toc -->
+## Optional vs. Required
+
+This extension is required, meaning it must be placed in both the `extensionsUsed` and `extensionsRequired` lists in the tileset JSON.
+
 <!-- omit in toc -->
 ## Contents
 
 - [Overview](#overview)
 - [Concepts](#concepts)
-  - [Metadata](#metadata)
+  - [Metadata Groups](#metadata-groups)
   - [Implicit Tiling](#implicit-tiling)
-- [Schema Updates](#schema-updates)
+    - [Metadata Groups in Implicit Tiling](#metadata-groups-in-implicit-tiling)
+- [Extension Schema Reference](#extension-schema-reference)
+- [`3DTILES_implicit_tiling` Subtree JSON Reference](#3dtiles_implicit_tiling-subtree-json-reference)
 
 
 ## Overview
 
 This extension adds support for multiple contents per tile.
 
-This is useful for datasets that have multiple content layers. Normally layering is achieved by combining contents into a [Composite](../../../specification/TileFormats/Composite/README.md) content, or by placing contents into sibling tiles, or by creating separate tilesets. With this extension content layers can exist cleanly in the same tileset, while allowing contents to be requested independently from each other.
+Contents can be organized into **groups**. For example, each tile could store two different representations of the same data using two contents: a point cloud and a triangle mesh, each representing the same terrain:
+
+<img src="figures/positional-groups.jpg" width="500" />
+
+When this extension is combined with [`3DTILES_metadata`](../../3DTILES_metadata/README.md), arbitrary groups of contents are supported. Each group can also have metadata associated with it.
+
+<img src="figures/metadata-groups.jpg" width="500" />
```

Comment:
Is this diagram supposed to show the cars?

Reply:
ah right, I made the diagrams separately, but in this order it makes sense to have the cars.